lms/djangoapps/instructor/access.py

```diff
@@ -72,6 +72,7 @@ def _change_access(course, user, level, action, send_email=True):
 
     NOTE: will create a group if it does not yet exist.
     """
+    from lms.djangoapps.ccx.models import CustomCourseForEdX
```

Comment:
why import here?

Reply:
good catch!, moved to the top of file,
I realize also will cause calls to that function to take longer.